corehq/apps/hqwebapp/management/commands/build_requirejs.py

```diff
@@ -73,6 +74,9 @@ def handle(self, **options):
             # and pass in the file contents, since get_hash does another read.
             file_hash = self.get_hash(filename)
 
+            if module['name'] == "users/js/bundle":
```

Comment:
Would it make sense to include a comment here explaining why this was added? It seems like the sort of thing that might get stuck in here and no one knows if it can be removed or not.

Reply:
Added a revert commit in https://github.com/dimagi/commcare-hq/pull/28455, does that work?

Reply:
Yes, that's perfect.